templates/template.rb

```diff
@@ -38,6 +38,7 @@ class NodeParam < Param
   include KindTypes
 
   def rbs_class = "Node"
+  def sorbet_class = "YARP::Node"
```

Comment:
I'm not a big fan of adding the `sorbet_class` methods here, because they're so specific to that one file. Honestly I would much prefer a big ERB block at the beginning of the template file to handle this kind of thing.

I would take advantage of the `kind` field here, so that you have something more specific. It should make working with it easier.